tools/test.py

```diff
@@ -1527,6 +1527,7 @@ def Main():
         vmArch = archEngineContext.stdout.rstrip()
         if archEngineContext.exit_code is not 0 or vmArch == "undefined":
           print "Can't determine the arch of: '%s'" % vm
+          print archEngineContext.stderr.rstrip()
```

Comment:
Should be in a separate PR. Sometimes the test runner will swallow startup errors here.